app.rb

```diff
@@ -38,9 +39,8 @@
 VALID_COMMANDS = (Spotify.public_methods - Object.public_methods).freeze
 
 def process_spotify_command(args)
-  args.downcase!
   command, *params = *split_args(args)
```

Comment:
no way!!!!!! a splat!

Reply:
Yeah, this is a good example of the `first`, `rest` conversation we had before. I really wish Ruby had these methods built in, but since it doesn't, this is the next best thing.